lib/caasp_controller.pm

```diff
@@ -8,7 +8,10 @@ use lockapi 'barrier_destroy';
 use mmapi 'wait_for_children';
 
 use Exporter 'import';
-our @EXPORT = qw(confirm_insecure_https velum_login switch_to download_kubeconfig click_click xy);
+our @EXPORT = qw($admin
+  confirm_insecure_https velum_login switch_to download_kubeconfig click_click xy);
+
+our $admin = 'admin.openqa.test';
```

Comment:
Why not to declare is as a constant? I also believe that we can come up with better naming e.g. ADMIN_URL

Reply:
Constants don't interpolate, so usage would be less clean, compare:
```perl
script_run "ssh $admin 'reboot'";
script_run "ssh " . admin . " 'reboot'";
```

I kept original name because `ssh $admin_url` looked strange - it's not url.

Reply:
I've only saw a usage where we use this hostname to open link in Mozilla, so then maybe admin_node or admin_node_hostname. I guess you know better.
As per interpolation, there is @{[...]} trick, but for some it may be even more confusing than concatenation. As value is not supposed to change, IMHO it fits better.

Reply:
Renamed to `admin_fqdn`.
I would prefer to avoid constant - it fits better, but only complicates usage in this case.

Reply:
Done, thanks for review